Cargo.toml

```diff
@@ -175,6 +175,8 @@ tonic = { version = "0.5.2", default-features = false, features = [
   "transport",
   "tls",
 ] }
+prost = "0.8"
```

Comment:
do we need those unconditionally, or can they be also only included when activating the `grpc` feature?